platform/server-bundle/src/main/resources/webapp/WEB-INF/jsp/changePassword.jsp

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html ng-app="motech-dashboard">
+    <head>
+        <meta charset="UTF-8">
+        <title>MOTECH - Mobile Technology for Community Health</title>
+
+        ${mainHeader}
+
+    </head>
+
+    <body ng-show="ready" ng-controller="MotechMasterCtrl" class="body-down" ng-init="initChangePasswordViewData()">
```

Comment:
Another minor issue with the validation is an issue we have in a couple of places, and goes against good UI design - I failed even before I started. The error for passwords not matching appears even before I start typing in the confirmation.